### ISSUE TITLE
Using ++HTML+++ overrides +++IMAGE+++ relationships in document.xml.rels

### ISSUE THREAD

**Issue by Waigie** [guigrpa/docx-templates#44]
I'm trying to create a report from template which contains a `+++HTML [...]+++` command as well as a `+++IMAGE [...] +++` command. Both commands work correct on their own but the HTML command seems to remove all relationship entries in the `document.xml.rels` file for any image. The image itself can be found in the created report in the word/media directory.

Minimal template file to reproduce the issue: [template.docx](https://github.com/guigrpa/docx-templates/files/2565372/template.docx)
Created report using the above template file: [report.docx](https://github.com/guigrpa/docx-templates/files/2565371/report.docx)
Code to create the report:
```javascript
const report = await createReport({
   template: this.base64ToArrayBuffer(fileData),
   additionalJsContext: {
      sigImage: async (dataUrl) => {
          const resp = await fetch(dataUrl);
          const buffer = resp.arrayBuffer
              ? await resp.arrayBuffer()
              : await resp.buffer();
           return { width: 6, height: 2, data: buffer, extension: '.png' };
       },
   },
   data: () => {
      return {
         data
      }
   }
});
```
Data to create the report:
```json
{
    signatur: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABQgA[...]"
    targets: "<ul><li>Test</li><li>Test</li><li>Test</li></ul>"
}
```

**Comment by gdeliaesgine** [comment]
I am having the same issue here. Can it be fixed?

**Comment by johannordin** [comment]
Yeah me to. :(

**Comment by vdechef** [comment]
Sorry, I don't have much free time these days. I will try to investigate this problem next week.

**Comment by guigrpa** [comment]
Thanks for the help, @vdechef!

**Comment by johannordin** [comment]
@Waigie could you elaborate? I dont quite get it :)

**Comment by guigrpa** [comment]
Hi there! Could you please check v2.9.0-rc.0? It should solve this issue…

**Comment by vdechef** [comment]
The problem occurs also with +++LINK+++
It happens because we forgot to use `await `when calling `processHtml`, `processImage`, and `processLink`
I fixed this, but I still have an issue with a corrupted file (maybe another problem with promises elsewhere).

**Comment by guigrpa** [comment]
I think the issue mentioned by @vdechef is related. Could you please install v2.9.0-rc.0 and tell me if it solves your use cases?

**Comment by guigrpa** [comment]
Btw, @vdechef: you identified part of the problem (missing `await`s in those functions), but there's more: when you read a file, JSZip doesn't give you the updated version of a file if you've included it in the zip file from a previous section of your code. This is, in principle, solved in v2.9.0-rc.0.

**Comment by vdechef** [comment]
#47
